### PR TITLE
JDBC_Intro_HW

### PR DESCRIPTION
**Pull request by AlexPravda1**
- fixed Injector
- implemented Manufacturer Dao methods
- main methods checks are written to logs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ But let's take one step at a time: in the current course, you will take care of
 
 - Establish connection to your Database.
 - Create `init_db.sql` file in `src/main/resources` folder.
-- Create `Manufacturer` model.
-- Create DAO  layer for `Manufacturer` model. Below you can see the list of required methods.
+- Create `Manufacturer` mate.jdbc.model.
```
Comment (RomanDubovskyi) [inline README.md:17]: don't change readme

File: src/main/java/jdbc/hw/dao/ManufacturerDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            createMfStatement.setString(2, manufacturer.getCountry());
+            createMfStatement.executeUpdate();
+            ResultSet generatedKeys = createMfStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create new entry to [manufacturers] "
+                        + "SQL database for manufacturer name " + manufacturer.getName()
```
Comment (RomanDubovskyi) [inline src/main/java/jdbc/hw/dao/ManufacturerDaoImpl.java:47]: it's better to add entire object to message. This way we won't need to change this code if we edit any fields

File: src/main/java/jdbc/hw/dao/ManufacturerDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+@Log4j2
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String CREATE_REQ =
+            "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
+    private static final String MF_SELECT_BY_ID_REQ =
+            "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false";
+    private static final String ALL_MF_REQ =
+            "SELECT * FROM manufacturers WHERE is_deleted = false";
+    private static final String UPDATE_MF_REQ =
+            "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?";
```
Comment (RomanDubovskyi) [inline src/main/java/jdbc/hw/dao/ManufacturerDaoImpl.java:26]: let's add is_deleted check

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlexPravda1) [reply README.md:17]: apologize, it was not intended. Assume misclick during commit and push

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request closed without merging**